contrib/clojure-package/examples/cnn-text-classification/src/cnn_text_classification/data_helper.clj

```diff
@@ -190,6 +198,8 @@
         vocab-embeddings (case pretrained-embedding
                            :glove (->> (load-glove (glove-file-path embedding-size))
                                        (build-vocab-embeddings vocab embedding-size))
+                           :fastText (->> (load-fasttext fasttext-file-path)
```

Comment:
I would prefer a keyword like `:fast-text` or `:fasttext` instead of having an uppercase character.